mp4ra_FFV1.md

```diff
@@ -0,0 +1,34 @@
+# Mapping FFV1 into the MP4
+
+This document represents preparation notes for a request to mp4ra.org to register FFV1 for use in MP4, following the instructions at the [mp4ra.org request page](http://www.mp4ra.org/request.html).
+
+## Email Request
+
+Draft of email.
+
+### For registration with ISO
+to: mp4reg@group.apple.com
+subject: mp4reg FFV1
+
+### For registration with QuickTime
+to: qtfourcc@group.apple.com for registration with QuickTime
+subject: qtreg FFV1
+
+cc: ffmpeg-devel?
+
+### Email Message
+
+[Name] of [address] at [organization url] requests the registration of an identifier for a lossless video encoding, FFV1, which is presently under standardization within the IETF [cellar working group](https://datatracker.ietf.org/wg/cellar/charter/).
+
+This request is for a codec kind of codepoint with a video stream type. The recommended identifier is "FFV1". The specification was initially published by FFmpeg at http://ffmpeg.org/~michael/ffv1.html. Presently the specification is under development by the IETF cellar working group and the most up-to-date copies of the specification are available at https://datatracker.ietf.org/doc/draft-niedermayer-cellar-ffv1/ (possibly changing to https://datatracker.ietf.org/doc/draft-ietf-cellar-ffv1/ later). Development work on the specification may be viewed at https://github.com/FFmpeg/FFV1.
+
+The codec could be briefly described as: "FFV1, a lossless intra-frame video codec".
+
+An authorized representation for the code-point may be contacted at:
+  cellar listserv? ffmpeg-devel? [they request contact name, title, org, email, phone]
+
+The date of the definition or implementation could be set to June 9, 2003 when FFV1 was initially released. Or August 3, 2013 when the latest version of FFV1 was released. A date for the official release of the specification for FFV1 from the IETF is not yet known, though earlier version of the specification were released by FFmpeg as recently as 2013.
+
+After registration, we intent to collaborate with the developers of FFV1 codecs to implement the registered identifier when FFV1 is used.
```

Comment:
Actually already the current situation, so I would better say that this is making official the current situation.